src/framework/pages/admin/AbstractMainMenu.java

```diff
@@ -20,6 +21,7 @@
 public abstract class AbstractMainMenu {
 	protected WebDriver driver;	
 	protected WebDriverWait wait;
+	UIMethods uiMethod = new UIMethods();
```

Comment:
Be careful this is incorrect, please check your changes and do the process (commit -> pull ->  commit - > push) again!!!!